otbox.py

```diff
@@ -206,52 +399,69 @@ def _excecute_command_safely(self, deviceType, deviceId, payload, cmd):
 
     def _mqtt_handler_echo(self, deviceType, deviceId, payload):
         '''
-        opentestbed/deviceType/box/deviceId/box1/cmd/echo
+        {{TESTBED}}/deviceType/box/deviceId/box1/cmd/echo
         '''
         assert deviceType==DEVICETYPE_BOX
 
         return json.loads(payload)
 
     def _mqtt_handler_status(self, deviceType, deviceId, payload):
         '''
-        opentestbed/deviceType/box/deviceId/box1/cmd/status
+        {{TESTBED}}/deviceType/box/deviceId/box1/cmd/status
         '''
         assert deviceType==DEVICETYPE_BOX
 
+        # Checks if `hostname -I` is a valid command
+        try:
+            ip_addresses = subprocess.check_output(["hostname", "-I"])
+        except subprocess.CalledProcessError:
+            ip_addresses = subprocess.check_output(["hostname", "-i"])
+
+        ip_addresses =ip_addresses.decode('utf8').rstrip()
+
         returnVal       = {
-            'software_version':   OTBOX_VERSION,
-            'currenttime':        time.ctime(),
-            'starttime':          time.ctime(self.start_time),
-            'uptime':             '{0}'.format(datetime.timedelta(seconds=(time.time()-self.start_time))),
-            'motes':              self.motesinfo,
-            'IP_address':         subprocess.check_output(["hostname","-I"]).rstrip(),
-            'location':           self.location,
-        }
+                'software_version':   OTBOX_VERSION,
+                'currenttime':        time.ctime(),
+                'starttime':          time.ctime(self.start_time),
+                'uptime':             '{0}'.format(datetime.timedelta(seconds=(time.time()-self.start_time))),
+                'motes':              self.motesinfo,
+            'IP_address': ip_addresses,
+            'host_name': self.OTBOX_ID,
```

Comment:
indention issue. 
Could you explain why this is needed? The API needs to update if we add host_name in the returnVal as well.

Reply:
As OpenTestbed is deployed dynamically at w-iLab.t, it is nice to have some extra info in the status-call. If this breaks things for you, we can revert this.